### ISSUE TITLE
The new API keys doesn't work?

### ISSUE THREAD

**Issue by Zyymotic** [Esri/arcgis-maps-sdk-unity-samples#154]
Hello great developers.

I am sorry to bother you, because the fault is guaranteed on my end. I recently got access to generate API keys in ESRI, for work on a project at my University. We we're planning on implementning features from your awesome SDK to an AR app. But we just can't seem to get the API key working. I can see that you recently updated the API key check in the samples to check for "New API key" which is 252 characters long. But the ones i get from ESRI is either 272 or 296 characters long.

When scouting the auth and security site on ESRI it has a fine table with the current working SDK's with the API's and Unity SDK was not one of them. Is there an overlap in development right now and is it expected to work in the future? Or is it working, but not for me?

I do realise that this might be outside the scope of your awesome work, but we are quite sad to see this opportunity wash away if the API doesn't work.

TLDR;
My API key is longer than yours. How do i get one that works with your SDK

I hope you can give an explanation at least and i do not expect you to solve it. Best regards
A frustrated study group.

**Comment by ZackAllen** [comment]
https://github.com/Esri/arcgis-maps-sdk-unity-samples/commit/6a931495b4331103566b0a16bab644719a99aac8

We have removed the length check. We were previously able to rely on it, but the key generation changed.
Apologies for the inconvenience.
